test/local/rns.available.test.ts

```diff
@@ -21,14 +22,18 @@ import { Options } from '../../src/types';
 import { labelhash } from '../../src/utils';
 import { ZERO_ADDRESS } from '../../src/constants';
 
+const web3Instance = web3 as unknown as Web3;
+const rsk3 = new Rsk3(web3.currentProvider);
```

Comment:
Also, is there a way to initialise rsk3 **without** referencing web3?
That would be ideal for detecting feature parity.

Reply:
We are using https://docs.openzeppelin.com/test-environment/0.1/ to test, it exposes a `web3` instance with the unlocked accounts already set. I did not find any other way to set the `Rsk3` instance with those accounts